.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - "2.7"
+install: pip install nbmolviz
```

Comment:
Note that this command will install the latest release from PyPI - I think you might want `pip install .` here to install from the github version